public/docs/_examples/toh-6/ts/app/heroes.component.css

```diff
@@ -57,3 +58,11 @@ button {
 button:hover {
   background-color: #cfd8dc;
 }
+/* #docregion delete-hero */
```

Comment:
Rename the region to "additions" (since you'll be re-introducing the `.error` style that was inadvertently dropped --- see my comment on the original `sample.css`), and make the corresponding update in the `.jade`.